### PR TITLE
Feature/quicklink rubiks

### PR DESCRIPTION
**Pull request by JonathanDuvalV**
Je n'étais pas certains pour l'icon, j'ai choisi celui la
image

### CODE REVIEW

File: lib/l10n/intl_fr.arb
Comment (camillebrulotte) [inline lib/l10n/intl_fr.arb:129]: Leur site ne semble pas avoir de "s" dans leur nom. Faurais peut-être l'enlever ? 

http://rubik.clubnaova.ca/

**Comment by apomalyn** [comment]
On hold until #57 is merged

File: lib/l10n/intl_en.arb
Comment (AlexandreMarkus) [inline lib/l10n/intl_en.arb:129]:
```suggestion
  "ets_schedule_generator": "Rubik",
```

File: lib/l10n/intl_fr.arb
Comment (AlexandreMarkus) [inline lib/l10n/intl_fr.arb:129]:
```suggestion
  "ets_schedule_generator": "Rubik",
```

**Comment by apomalyn** [comment]
> DO NOT MERGE!
> On hold until #57 is merged, I will review it later today or tomorrow 😄

#57 is merged! Will merge this PR when approved.

**Review by apomalyn — CHANGES_REQUESTED** [review]
Please pull master into your branch :smiley: otherwise I don't have anything to add :+1:

**Comment by JonathanDuvalV** [comment]
> Please pull master into your branch 😃 otherwise I don't have anything to add 👍

Sorry about that 😅. I was busy in the last few days. Merged master into the branch and fixed to use correct title for the rubik link.

**Comment by JonathanDuvalV** [comment]
Also it looks like Prettier changed the formating on the quicklinks file. Sorry about that

**Review by apomalyn — APPROVED** [review]
LGTM! 🍾 🎉

**Pull request merged**
